tools/docker/windows/install_intellij.sh

```diff
@@ -0,0 +1,46 @@
+#!bin/bash
+
+ed=C
+
+# Fetch the most recent version
+VERSION=$(wget "https://www.jetbrains.com/intellij-repository/releases" -qO- | grep -P -o -m 1 "(?<=https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/idea/BUILD/)[^/]+(?=/)")
+
+# Prepend base URL for download
+URL="https://download.jetbrains.com/idea/ideaI$ed-$VERSION.tar.gz"
+
+echo $URL
+
+# Truncate filename
+FILE=$(basename ${URL})
```

Comment:
1. Quotes outside, quotes inside, drop `{}`s, but
2. This is not a path so don't use `basename`, instead just throw everything up to the last `/` (eg, `${foo##*/}`).